tests/Resources/Page/OCPDashboard/OperatorHub/OperatorHub.robot

```diff
@@ -37,9 +42,16 @@ Confirm Show Operator
    Click Element    //*[@id="confirm-action"]
 
 Wait Until Installation Completes
-    Wait Until Page Contains    ready for use   timeout=150
+    Wait Until Page Contains    ready for use   timeout=300
 
 Operator Should Be Installed
     [Arguments]    ${operator}
     Page Should Contain    ${operator}
     Page Should Contain    ready for use
+
+Get List Of Operator Available
```

Comment:
"Get List [..]" but it returns a numeric value

Reply:
good catch ..my intention is to get the number ...will modify it

Reply:
may help to returns the list of web elements?

Reply:
my intention to have this keyword to provide the no of operator present with same name so that we can verify it if somewhere in future if removes we will get to know

Reply:
yeah I got the intention. It's fine